french_(france)/global.ini

```diff
@@ -64678,7 +64678,7 @@ text_ui_units_EVA,P=EVA
 text_ui_units_Temp,P=Temp
 text_ui_units_angle=º
 text_ui_units_bpm=bpm
-text_ui_units_colon=:
```

Comment:
Does the space is needed here?

Reply:
a priori je dirais oui (tjs non breaking)